src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntityOceanPumper.java

```diff
@@ -191,19 +210,15 @@ public void update() {
 
         if (getWorld().isRemote || getOffsetTimer() % 20 != 0 || !isStructureFormed()) return;
```

Comment:
this would somehow crash the game if the ocean pumper is placed. update() would need some intensive rework.



```
[21:56:40] [Server thread/ERROR] [net.minecraft.server.MinecraftServer]: Encountered an unexpected exception
net.minecraft.util.ReportedException: Ticking block entity
	at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:847) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:742) ~[MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:192) ~[IntegratedServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:591) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_382]
Caused by: java.lang.NullPointerException
	at supersymmetry.common.metatileentities.multi.electric.MetaTileEntityOceanPumper.getEnergyConsumedPerPump(MetaTileEntityOceanPumper.java:151) ~[MetaTileEntityOceanPumper.class:?]
	at supersymmetry.common.metatileentities.multi.electric.MetaTileEntityOceanPumper.drainEnergy(MetaTileEntityOceanPumper.java:155) ~[MetaTileEntityOceanPumper.class:?]
	at supersymmetry.common.metatileentities.multi.electric.MetaTileEntityOceanPumper.update(MetaTileEntityOceanPumper.java:212) ~[MetaTileEntityOceanPumper.class:?]
	at gregtech.api.metatileentity.MetaTileEntityHolder.update(MetaTileEntityHolder.java:193) ~[MetaTileEntityHolder.class:?]
	at net.minecraft.world.World.updateEntities(World.java:2007) ~[World.class:?]
	at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:642) ~[WorldServer.class:?]
	at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:841) ~[MinecraftServer.class:?]
	... 4 more```